### PR TITLE
phi0474.phi003

### PR DESCRIPTION
**Pull request by lcerrato**
merge changes #109

### CODE REVIEW

**Comment by lcerrato** [comment]
@PonteIneptique May I impose?

**Comment by PonteIneptique** [comment]
Please, of course :+1:

File: data/phi0474/phi003/phi0474.phi003.perseus-eng2.xml [hunk 1]
```diff
@@ -159,18 +121,22 @@ Added to repository. Passes TEI parsing.
...
-          </argument>
+          </div>
         </front>
-        <body>
-          <milestone n="1" unit="chapter"/>
-          <milestone n="1" unit="section"/>
+    
+    
+          <body>
+          <div type="edition" xml:lang="eng" n="urn:cts:latinLit:phi0474.phi003.perseus-eng2">
```
Comment (PonteIneptique) [inline data/phi0474/phi003/phi0474.phi003.perseus-eng2.xml]: This should be @type="translation"

**Comment by lcerrato** [comment]
fixed!

**Pull request merged**